generated.nix

```diff
@@ -1,10 +1,14 @@
 # this file is autogenerated by .github/workflows/update.yml
 {
-  url = "https://github.com/nix-community/nix-index-database/releases/download/2024-12-22-031612/index-";
+  url = "https://github.com/gvolpe/nix-index-database/releases/download/2024-12-22-163440/index-";
```

Comment:
I intentionally left this file changes here so that my fork continues to work. If this is merged, this can be updated by running the `update` workflow manually the first time.